area_calc.py

```diff
@@ -2,43 +2,50 @@
 
 
 class Calc():
-    if shape == 'TRAPEZOID': base2 = int(input('Enter second base size:'))
-
-    def __init__(self, height, base):
-        self.height = height
-        self.base = base
-        base = int(input('Enter base:'))
-        height = int(input('Enter height:'))
+#    if shape == 'TRAPEZOID': base2 = int(input('Enter second base size:'))
 
+    def __init__(self):
+        self.base = int(input('Enter base:'))
+        self.height = int(input('Enter height:'))
 
 class Square(Calc):
 
-    def __init__(self, height, base):
-        super().__init__(height, base)
-
     def getArea(self):
         return self.base * self.height
 
 
 class Triangle(Calc):
 
-    def cal(self):
-        return print('Calculated triangle area:', Calc.base * Calc.height / 2)
+    def getArea(self):
+        return self.base * self.height / 2
 
 
 class Rectangle(Calc):
 
-    def cal(self):
-        return print('Calculated rectangle area:', Calc.base * Calc.height)
+    def getArea(self):
+        return self.base * self.height
 
 
 class Trapezoid(Calc):
 
-    def cal(self):
-        return print('Calculated trapezoid area:', (Calc.base + Calc.base2) * Calc.height / 2)
+     def __init__(self, height, base, base2):
```

Comment:
override constructor of base class (pay attention to  prototype of  constructor )